brightchain-lib/src/lib/blockMetadata.ts

```
@@ -49,11 +39,24 @@
       size: this.size,
       type: this.type,
       dataType: this.dataType,
-      originalDataLength: this.lengthWithoutPadding,
+      lengthWithoutPadding: this.lengthWithoutPadding,
       dateCreated: this.dateCreated,
     });
   }
 
+  public static fromJsonValidator(data: any): void {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)